### PR TITLE
Add command to create admin account

### LINKED ISSUE #410: Setup default account
**Is your feature request related to a problem? Please describe.**
Currently there is no default account which is created while bootstrapping the database for security reasons. Come up with a better way instead of using default credentials.

### PR DESCRIPTION
**Pull request by maxammann**
fixes #410

### CODE REVIEW

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt [hunk 1]
```diff
@@ -7,35 +7,56 @@ import app.ehrenamtskarte.backend.stores.importer.DataImporter
...
+import com.github.ajalt.clikt.parameters.arguments.argument
 import java.io.File
 
-
 class Entry : CliktCommand() {
-    private val exportApi by option(help = "Export GraphQL schema. Given ")
-    private val importFlag by option("--import").flag()
+    override fun run() {
+    }
+}
```
Comment (klinzo) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt:17]: Could be deleted right?

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt [hunk 2]
```diff
@@ -7,35 +7,56 @@ import app.ehrenamtskarte.backend.stores.importer.DataImporter
...
+    override fun run() {
+        val production = System.getProperty("app.production", "").isNotEmpty()
+        Database.setup(!production)
+        Database.createAccount(email, password)
+    }
+}
+
+class Execute : CliktCommand(help = "Starts the webserver") {
+    override fun run() {
+        val production = System.getProperty("app.production", "").isNotEmpty()
```
Comment (klinzo) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt:56]: Could this be generalized? Three usages of this

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt [hunk 3]
```diff
@@ -7,35 +7,56 @@ import app.ehrenamtskarte.backend.stores.importer.DataImporter
...
-            Database.setup(!production)
-            if (importFlag) {
-                println("Importing data from Lbe")
-                if (!DataImporter.import(!production)) {
-                    throw ProgramResult(statusCode = 5)
-                }
-            } else {
-                WebService().start(production)
-            }
+class Import : CliktCommand(help = "Imports stores from a configured data source. Various properties must be set.") {
```
Comment (klinzo) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt:28]: Upon executing `Run Backend with Import` I get:
> 15:05:45: Executing 'run run --args="--import"'...
Error: no such option: "--import"

File: docs/development-setup.md [hunk 4]
```diff
@@ -24,23 +24,22 @@ This short guide focuses on setting up the project using IntelliJ instead of And
...
    |Password|postgres|
    |Database|ehrenamtskarte|
-4. Install JDK8
-5. Run the backend: `cd backend && ./gradlew run` or `.\backend\gradlew.bat run` on Windows
-6. Take a look at the martin endpoints: [http://localhost:5002/tiles/accepting_stores/index.json](http://localhost:5002/tiles/accepting_stores/index.json) and [http://localhost:5002/tiles/accepting_stores/rpc/index.json](http://localhost:5002/tiles/accepting_stores/rpc/index.json). The data shown on the map is fetched from a hardcoded url and is not using the data from the local martin!
-7. Take a look at the style by viewing the test map: [http://localhost:5002](http://localhost:5002)
-8. Take a look at the backend: [http://localhost:7000](http://localhost:7000) (The public version is available at api.ehrenamtskarte.app)
+6. Install JDK8
+7. Run the backend: `cd backend && ./gradlew run --args="execute"` or `.\backend\gradlew.bat run --args="execute"` on Windows
+8. Create an admin account using `./gradlew run --args="create-admin <email> <password>"`
```
Comment (klinzo) [inline docs/development-setup.md:39]: Does it make sense to create an idea run config for this as well with a dummy admin account?

Comment (klinzo) [reply docs/development-setup.md:39]: Upon executing `./gradlew run --args="create-admin asfd@asdf.de 1234"` I get:
> INSERT INTO categories ("id", "name")
VALUES (8, 'Essen/Trinken/Gastronomie') ON CONFLICT ("id")
    DO UPDATE SET
        "name"='Essen/Trinken/Gastronomie';
INSERT INTO categories ("id", "name")
VALUES (9, 'Sonstiges') ON CONFLICT ("id")
    DO UPDATE SET
        "name"='Sonstiges';
Exception in thread "main" java.lang.IllegalStateException: No transaction in context.
        at org.jetbrains.exposed.sql.transactions.TransactionManager$Companion.current(TransactionApi.kt:98)
        at org.jetbrains.exposed.dao.EntityClass.new(EntityClass.kt:237)
        at org.jetbrains.exposed.dao.EntityClass.new(EntityClass.kt:220)
        at app.ehrenamtskarte.backend.auth.database.repos.AdministratorsRepository.insert(AdministratorsRepository.kt:18)
        at app.ehrenamtskarte.backend.common.database.Database$Companion.createAccount(Database.kt:35)
        at app.ehrenamtskarte.backend.CreateAdmin.run(EntryPoint.kt:50)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:171)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:180)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:16)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:395)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:392)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:410)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:435)
        at app.ehrenamtskarte.backend.EntryPointKt.main(EntryPoint.kt:62)
FAILURE: Build failed with an exception.

**Review by Schedulaar — APPROVED** [review]

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt (hunk 3 above)
Comment (maxammann) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt:28]: Thanks!

**Pull request merged**